src/util/MapPolyfill.ts

```diff
@@ -20,7 +20,7 @@ export class MapPolyfill {
     return this;
   }
 
-  delete(key): boolean {
+  delete(key: any) {
```

Comment:
Yes it does, but most of cases we're adding explicit return types but in here was removing. Was curious specific reason to remove which already exists.